packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -2437,12 +2434,18 @@ msgstr "Název ve výchozím jazyce není vyplněn"
 msgid "Name in the corresponding locale must be filled-in in order to display the file on the storefront"
 msgstr "Pro zobrazení souboru na doméně musí být vyplněn název v odpovídajícím jazyce"
 
+msgid "Name prefix"
+msgstr "Prefix názvu"
+
 msgid "Name serves only for internal use within the administration"
 msgstr "Název slouží pouze pro interní použití v rámci administrace"
 
 msgid "Name serves only for internal use within the administration."
 msgstr "Název slouží pouze pro interní použití v rámci administrace."
 
+msgid "Name suffix"
+msgstr "Suffix názvu"
```

Comment:
in czech it's with just one `f` - `Sufix`

Reply:
:+1: 733d4f3484